tasks/internal_reconcile_report_orphan/src/use_cases/adapter_interfaces/storage.py

```diff
@@ -10,5 +10,5 @@ class OrphansPageStorageInterface:
 
     def get_orphans_page(self,
                          orphan_record_filter: OrphanRecordFilter,
-                         logger: logging.Logger
+                         logger: Logger()
```

Comment:
Same question on Logger typehint.

Reply:
to initiate the logger. Doing this here directly.
logger = Logger()